authors/modmuss50.json

```diff
@@ -0,0 +1 @@
+{"name":"modmuss50","contacts":{"githubUrl":"https://github.com/modmuss50","githubUrl":"https://github.com/TechReborn/TechReborn"},"mods":[{"minecraftVersions":[{"modVersions":[{"modVersion":"1.16-3.4.5+build.88","modUrl":"https://github.com/TechReborn/TechReborn/releases/download/TechReborn-1.16-3.4.5%2Bbuild.88/TechReborn-1.16-3.4.5.build.88.jar"}],"minecraftVersion":"1.16.1 - 1.16"},{"modVersions":[{"modVersion":"1.15-3.3.11+build.224","modUrl":"https://github.com/TechReborn/TechReborn/releases/download/TechReborn-1.15-3.3.11%2Bbuild.224/TechReborn-1.15-3.3.11.build.224.jar"}],"minecraftVersion":"1.15 - 1.15.2"}],"minecraftVersion":"1.16"},{"modVersions":[{"modVersion":"1.14.4-3.0.22+build.273","modUrl":"https://github.com/TechReborn/TechReborn/releases?after=TechReborn-1.14.4-3.0.22%2Bbuild.273/TechReborn-1.14.4-3.0.22+build.273.jar"}],"minecraftVersion":"1.14.4"},{"modVersions":[{"modVersion":"1.12.2-2.27.3.1084","modUrl":"https://github.com/TechReborn/TechReborn/releases/download/TechReborn-1.12.2-2.27.1084/TechReborn-1.12.2-2.27.1084-universal.jar"}],"name":"Tech Reborn","description":"Tech Reborn is a completely standalone tech mod including tools and machines to gather resources, process materials, and progress through the mod.","sources":"https://github.com/TechReborn/TechReborn","iconUrl":"https://github.com/TechReborn/TechReborn/blob/1.16/src/main/resources/assets/techreborn/icon.png","id":"tech reborn"}]}
```

Comment:
I would also like to point out there is a few errors in this particular file. I would advise using a json formatter if you are not using one and icons aren't in raw so if you have opened FMM recently you might have experienced a crash. Try .cache to fix this